<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="main_layout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="left_layout">
        <item>
         <widget class="QGroupBox" name="apprentice_groupBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>170</height>
           </size>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="add_apprentice_button">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="edit_apprentice_button">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="delete_apprentice_button">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="year_edit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="time_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="profession_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="category_edit"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="time_edit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="first_name_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="first_name_edit"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="vacation_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="last_name_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="year_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="vacation_edit"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="last_name_edit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="table_groupBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="search_button">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="search_edit">
             <property name="minimumSize">
              <size>
               <width>260</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="combo_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="sort_comboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="apprentice_list"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_layout">
        <item>
         <widget class="QCalendarWidget" name="calendar">
          <property name="styleSheet">
           <string notr="true">/* Style for header area  ####################################################*/ 

QCalendarWidget QWidget {
	 alternate-background-color: #B8E2FF;
}

/* style for top navigation area ###############################################*/ 

#qt_calendar_navigationbar {
    background-color: #fff;
	/*border: 2px solid  #B8E2FF;*/
	border-bottom: 0px;
	/*border-top-left-radius: 5px;
	border-top-right-radius: 5px;*/
}

/* style for month change buttons ############################################ */

#qt_calendar_prevmonth, 
#qt_calendar_nextmonth {
	/* border delete */
    border: none;  
    /* delete default icons */
	qproperty-icon: none; 
	
    min-width: 13px;
    max-width: 13px;
    min-height: 13px;
    max-height: 13px;

    border-radius: 5px; 
	/* set background transparent */
    background-color: transparent; 
	padding: 5px;
}

/* style for pre month button ############################################ */

#qt_calendar_prevmonth {
	/* set text for button */
	/*qproperty-text: &quot;&gt;&quot;;*/
	margin-left:5px;
	image: url(:/icons/arrow_left.ico);
}

/* style for next month button ########################################### */
#qt_calendar_nextmonth {
	margin-right:5px;
	image: url(:/icons/arrow_right.ico);
    /* qproperty-text: &quot;&gt;&quot;; */
}
#qt_calendar_prevmonth:hover, 
#qt_calendar_nextmonth:hover {
    background-color: #55aaff;
}

#qt_calendar_prevmonth:pressed, 
#qt_calendar_nextmonth:pressed {
    background-color: rgba(235, 235, 235, 100);
}


/* Style for month and yeat buttons #################################### */

#qt_calendar_yearbutton {
    color: #000;
	margin:5px;
    border-radius: 5px;
	font-size: 13px;
	padding:0px 10px;
}

 #qt_calendar_monthbutton {
	width: 110px;
    color: #000;
	font-size: 13px;
	margin:5px 0px;
    border-radius: 5px;
	padding:0px 2px;
}

#qt_calendar_yearbutton:hover, 
#qt_calendar_monthbutton:hover {
    background-color: #55aaff;
}

#qt_calendar_yearbutton:pressed, 
#qt_calendar_monthbutton:pressed {
    background-color: rgba(235, 235, 235, 100);
}

/* Style for year input lineEdit ######################################*/

#qt_calendar_yearedit {
    min-width: 53px;
    color: #000;
    background: transparent;
	font-size: 13px;
}

/* Style for year change buttons ######################################*/

#qt_calendar_yearedit::up-button { 
	image: url(:/icons/arrow_top.ico);
    subcontrol-position: right;
}

#qt_calendar_yearedit::down-button { 
	image: url(:/icons/arrows/arrow_down.ico);
    subcontrol-position: left; 
}

#qt_calendar_yearedit::down-button, 
#qt_calendar_yearedit::up-button {
	width:10px;
	padding: 0px 5px;
	border-radius:3px;
}

#qt_calendar_yearedit::down-button:hover, 
#qt_calendar_yearedit::up-button:hover {
	background-color: #55aaff;
}

/* Style for month select menu ##################################### */

#calendarWidget QToolButton QMenu {
     background-color: white;

}
#calendarWidget QToolButton QMenu::item {
	/*padding: 10px;*/
}
 #calendarWidget QToolButton QMenu::item:selected:enabled {
    background-color: #55aaff;
}

#calendarWidget QToolButton::menu-indicator {
	/* Remove toolButton arrow */
      /*image: none; */
	nosubcontrol-origin: margin;
	subcontrol-position: right center;
	margin-top: 10px;
	width:20px;
}

/* Style for calendar table ########################################## */
#qt_calendar_calendarview {
	/* Remove the selected dashed box */
    outline: 0px;
	/*curved borders*/
	/*border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
	border: 2px solid  #B8E2FF;
	border-top: 0px;*/
	
}

#qt_calendar_calendarview::item:hover {
   border-radius:5px;
	background-color:#aaffff;
}

#qt_calendar_calendarview::item:selected {
    background-color: #55aa7f; 
	border-radius:5px;
}
</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="vacation_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sick_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="time_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="school_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="department_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="delete_event_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="edit_event_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="event_list"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
